src/main/java/com/base/util/BaseAppUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.base.util;
+
+import com.getbase.Client;
+import com.getbase.Configuration;
+import org.springframework.core.env.MissingRequiredPropertiesException;
+
+/**
+ * Created by przemek on 19.10.2016.
+ */
+public interface BaseAppUtil {
```

Comment:
don't use interfaces like that -> make it final concrete class, or use lombok's @UtilityClass